### PR TITLE
Add application of vetoes to plots

### PR DESCRIPTION
**Pull request by EthanMarx**

### CODE REVIEW

File: projects/sandbox/pyproject.toml [hunk 1]
```diff
@@ -262,3 +262,5 @@ log_file = "${base.basedir}/log/sensitive_volume.log"
 max_far = 1000
 sigma = 0.1
 verbose = "${base.verbose}"
+gate_paths = {"H1" = "/home/ethan.marx/public_html/forks/BBHNet/projects/sandbox/vizapp/vizapp/vetoes/H1-O3_GATES_1238166018-31197600.txt", "L1" = "/home/ethan.marx/public_html/forks/BBHNet/projects/sandbox/vizapp/vizapp/vetoes/L1-O3_GATES_1238166018-31197600.txt"}
```
Comment (alecgunny) [inline projects/sandbox/pyproject.toml:265]: how big are these files? Are they worth adding to version control?

**Review by alecgunny — APPROVED** [review]

File: projects/sandbox/pyproject.toml (hunk 1 above)
Comment (EthanMarx) [reply projects/sandbox/pyproject.toml:265]: They're versioned already in the vizapp project

Comment (EthanMarx) [reply projects/sandbox/pyproject.toml:265]: These files live in a git.ligo.org repo so maybe we can have the script download them and store them via an environment variable or something

Comment (EthanMarx) [reply projects/sandbox/pyproject.toml]: @alecgunny I added ability to apply vetoes to foreground events, and removed the paths to vetoes as arguments of the function in favor of pointing directly to the versioned control versions.

**Review by alecgunny — APPROVED** [review]
Looks great

**Pull request merged**